lib/etcdv3/etcdrpc/rpc_pb.rb

```diff
@@ -369,6 +404,7 @@
   add_enum "etcdserverpb.AlarmType" do
     value :NONE, 0
     value :NOSPACE, 1
+    value :CORRUPT, 2
```

Comment:
https://github.com/davissp14/etcdv3-ruby/blob/master/lib/etcdv3/maintenance.rb  Needs to be updated.